core/src/avm2/globals/flash/net/url_loader.rs

```diff
@@ -130,47 +50,15 @@ fn spawn_fetch<'gc>(
         .get_property(&QName::dynamic_name("url").into(), activation)?
         .coerce_to_string(activation)?;
 
+    let url = url.to_utf8_lossy();
```

Comment:
I think this change is a rebase conflict leftover, and as such can be avoided (unless I'm missing something).